gtsfm/common/sfm_track.py

```diff
@@ -69,6 +69,18 @@ def select_subset(self, idxs: List[int]) -> "SfmTrack2d":
 
         return SfmTrack2d(inlier_measurements)
 
+    def select_for_cameras(self, camera_idxs: Set[int]) -> "SfmTrack2d":
+        """Generates a new track with only those measurements which are in camera_idxs.
+
+        Unlike `select_subset`, this method does not require all camera_idxs to have a measurement in this track.
+
+        Returns:
+            Track with the subset of measurements.
+        """
+        measurements = [m for m in self.measurements if m.i in camera_idxs]
+
```

Comment:
nit: i propose we get rid of the newline between these two lines, to keep it a bit more compact

Reply:
Noting as resolved, thanks.